schemas/devContainer.base.schema.json

```diff
@@ -413,16 +413,14 @@
 						"gpu": {
 							"oneOf": [
 								{
-									"type": [
-										"boolean",
-										"string"
-									],
-									"enum": [
-										true,
-										false,
-										"optional"
-									],
-									"description": "Indicates whether a GPU is required. The string \"optional\" indicates that a GPU is optional. An object value can be used to configure more detailed requirements."
+									"type": "boolean",
+									"const" : true,
+									"description": "Indicates a GPU is required."
```

Comment:
The description can be shown in editors. I suggest we copy the original description to both descriptions here, so the user will always get the full description for all 3 cases.